src/commands/createWebApp/createWebApp.ts

```diff
@@ -14,6 +14,7 @@ export async function createWebApp(context: IActionContext & Partial<ICreateChil
     }
 
     const newSite: WebAppTreeItem = <WebAppTreeItem>await node.createChild(context);
+    await newSite.enableHttpLogs();
```

Comment:
This won't happen if people create from the blue up arrow, which is the more common case